Python/compile.c

```diff
@@ -8627,6 +8642,10 @@ assemble(struct compiler *c, int addNone)
         goto error;
     }
 
+#ifdef Py_DEBUG
+    assert_no_redundant_jumps(g);
```

Comment:
... then this becomes `assert(no_redundant_jumps())` and you can drop the `#ifdef Py_DEBUG`